pytket/phir/phirgen.py

```diff
@@ -140,6 +142,9 @@ def convert_subcmd(op: tk.Op, cmd: tk.Command) -> dict[str, Any]:
                 [arg_to_bit(cmd.bits[i]) for i in range(len(cmd.bits))], op.values
             )
 
+        case tk.CopyBitsOp():
+            return assign_cop([cmd.bits[0].reg_name], [cmd.args[0].reg_name])
```

Comment:
There's no requirement in pyket that all the bits belong to the same "register", so I think there should be a check here that all the register names in `cmd.bit` are the same, and similarly in `cmd.args` -- if not, we can throw an error (I don't think there's a need to support that case in phir).

Reply:
This is helpful, thanks. The API seems to be quite vague about a bunch of these questions. Those checks might need to be added in several other places then.

Do you know if `CopyBitsOp` is only used for `add_c_copyreg`? The translation here assumes that and hence drops the bit indices.

Reply:
It's used in `add_c_copybits` as well, where the bits can be arbitrary, for instance
```python
c.add_c_copybits([Bit("b", 2), Bit("a", 1)], [Bit("a", 0), Bit("b", 0)])
```